src/main/java/Messenger/MessengerNotification.java

```diff
@@ -0,0 +1,57 @@
+package Messenger;
+
+import Notification.Notification;
+
+import java.util.Date;
+
+
+/**
+ * Created by kenjik on 06.11.17.
+ */
+public class MessengerNotification implements Notification {
+
+    private String senderName;
+    private String receiverName;
+    private Date sendDate;
+    private String message;
+
+    public MessengerNotification(String senderName, String receiverName, Date sendDate, String message) {
```

Comment:
If you want to ensure clients will be using `MessengerNotificationBuilder` you should make this constructor package-private